chrome/content/zotero/xpcom/zotero.js

```diff
@@ -1162,6 +1154,30 @@ Components.utils.import("resource://gre/modules/osfile.jsm");
 		
 		return useProfileDir ? true : file;
 	}
+
+
+	this.checkDataDirectory = function (path){
+		if(Zotero.File.isDropboxDirectory(path)) {
+			let check = {value: false};
+			let index = Services.prompt.confirmEx(
+				null,
+				Zotero.getString('general.warning'),
+				Zotero.getString('dataDir.existingDirInDropbox.text', Zotero.appName),
+				Services.prompt.STD_YES_NO_BUTTONS,
+				null, null, null,
+				Zotero.getString('general.dontShowWarningAgain'),
+				check
+			);
+
+			// Yes - display dialog.
+			if (index == 0) {
+				Zotero.chooseZoteroDirectory(true);
+			}
+			if(check.value === true) {
```

Comment:
Can just be `if (check.value)` (and nit: space after `if` here and elsewhere — space after keyword, not after function name)